strum_macros/src/helpers/metadata.rs

```diff
@@ -0,0 +1,241 @@
+use proc_macro2::{Span, TokenStream};
```

Comment:
This whole file is very cool I have clearly not been keeping up with syn's features :)